Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Add cstdint includes to fix builds with gcc >= 13
Based on patch from Kuoi <kuoi@bioarchlinux.org>
---
diff --git a/auxprogs/homGeneMapping/include/sqliteDB.hh b/auxprogs/homGeneMapping/include/sqliteDB.hh
index e842bcf9..67338bef 100644
--- a/auxprogs/homGeneMapping/include/sqliteDB.hh
+++ b/auxprogs/homGeneMapping/include/sqliteDB.hh
@@ -13,6 +13,7 @@
 #include <string>
 #include <vector>
 #include <sqlite3.h>
+#include <cstdint>
 
 using namespace std;
 
diff --git a/include/sqliteDB.hh b/include/sqliteDB.hh
index e66374b9..67a37826 100644
--- a/include/sqliteDB.hh
+++ b/include/sqliteDB.hh
@@ -11,6 +11,7 @@
 #include <string>
 #include <vector>
 #include <sqlite3.h>
+#include <cstdint>
 
 using namespace std;
 
